spec/carrierwave/storage/aws_file_spec.rb

```diff
@@ -62,6 +63,16 @@
       aws_file.url
     end
 
+    let(:cloudfront_signer) { -> (unsigned_url, options) { [unsigned_url, signature].join('?') } }
```

Comment:
These `lets` are only used once, they may as well be put into the spec as variables.